TFNetworkLayer.py

```diff
@@ -2953,7 +2957,12 @@ def ceildiv(a, b):
 
   @classmethod
   def _get_out_type_from_opts(cls, n_out, filter_size, padding, strides=1, dilation_rate=1, sources=(),
-                              input_expand_dims=0, input_add_feature_dim=False, input_split_feature_dim=None, **kwargs):
+                              input_expand_dims=0, input_add_feature_dim=False, input_split_feature_dim=None,
+                              auto_use_channel_first=False,
+                              **kwargs):
+    data = get_concat_sources_data_template(sources)
+    auto_use_channel_first = auto_use_channel_first or data.is_batch_feature_major
+    # The output format is the same as the input. We maybe change to channel-first below.
```

Comment:
I think this comment is more confusing than helping. Maybe better just remove it.